website/src/content/development/layout-guidelines.md

```diff
@@ -0,0 +1,122 @@
+# Layout Guidelines
+
+Layouts are the foundation for interactive experiences. They affect the way data is structured and displayed. The different layout types set the minimum and maximum width of a container and its position on the page.
+
+### Design
+
+In commercetools, we always try to make better use of the space when building Merchant Center applications. As a result, we provide guidelines and rules that are recommended to be followed when developing applications, both by internal and external engineering teams. You can choose from 5 different layout options to display content in the Merchant Center:
+
+#### Narrow
+
+* Single Column narrow should be used for simple forms.
+* The form can be split into multiple sections by collapsible panels, cards, or headlines.
+
+IMAGE
+
+#### Wide (Single Column)
+* Single column wide should be used for content where more space is necessary.
+* It is a good fit for sections with complex functionality which don't need to be spread across the full width.
+* It can also be used for tables with a max of 5-7 columns when the possibility to add additional columns with the column manager is not given.
+
+IMAGE
+
+#### Wide (2/3 + 1/3)
+* This layout should be used on a page with a form. The right column can display general information or links about the page (like the meta dates, link to change history, link to product type, etc.).
+* The content on the right column is sticky on top. Only the 1st column is scrollable.
+
+IMAGE
+
+#### Wide (1/2 + 1/2)
+* This layout can be used for displaying relevant information next to each other to save vertical space.
+* It is important to group the content in each column, so it is clear to the user, that each column should be read/scanned vertically.
+
+IMAGE
+
+#### Full width
+* Full width should be used when the content has a big Data Table.
+* Full width can be also used in cases when it makes sense to group content horizontally.
+* Make sure that content in the full width layout is grouped in a way, that the reading flow is clear and the user knows how to scan through the page easily.
+
+IMAGE
+
+<details>
+<summary>Here you can see some examples</summary>
+
+![Screenshot 2023-01-16 at 12 25 53](https://user-images.githubusercontent.com/97907068/213112755-f547f4a2-e787-43a7-a6ed-6f09310d2301.png)
+
+</details>
+
+As not all combinations are valid there are also some restrictions on how these layouts can be combined on a single page.
+
+<details>
+<summary>Here you can see all valid combinations</summary>
+
+![Screenshot 2023-01-16 at 13 32 10](https://user-images.githubusercontent.com/97907068/213112785-c16eef58-b1e7-4dde-8244-48ff0ffbd7cd.png)
+
+</details>
+
+### Implementation
```

Comment:
Maybe we should add the technical docs to the [applications-components](https://docs.commercetools.com/custom-applications/api-reference/commercetools-frontend-application-components) section and just have a link here.
I can help with this.